_scripts/gen-faq-toc.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"regexp"
+	"strings"
+)
+
+func must(err error, fmtstr string, args ...interface{}) {
+	if err != nil {
+		log.Fatalf(fmtstr, args...)
+	}
+}
+
+func usage() {
+	fmt.Fprintf(os.Stderr, "gen-faq-to input-path output-path")
```

Comment:
Why not just use `os.Stderr`?

```suggestion
	os.Stderr.WriteString("gen-faq-to input-path output-path")
```